### ISSUE TITLE
Decide on a file system

### ISSUE THREAD

**Issue by veldyn** [Veldyn/2024-nas#1]

**Comment by veldyn** [comment]
1/19

Today we verified androidOS as our operating system, and discussed using NFS (https://www.ibm.com/docs/en/aix/7.1?topic=management-network-file-system) to read any file system from any other device. Ideally, this would allow us to integrate Windows, Mac, Linux, etc with any file system type we want.

We also discussed using GitHub as a repo to store final instructions, documentation, any VM files for virtualizing the operating system, and system configuration files.

For the file system we're going to investigate more into NFS to see how that is to implement. We talked about potentially buying a Raspberry Pi to load androidOS onto that and try to add NFS directly on that, rather than having to invest in all the hardware for the NAS and do it through that. There are concerns we have about Raid configuration with NFS, but we need to investigate that more.

If we can't get NFS working or encounter issues with that we'll use ExFAT, but we're already managed to find a Microsoft-supported guide for setup so we don't suspect this will be an issue.

We're trying to delay the purchase of hardware for some time, so we're looking to do research and perhaps look into virtualization for the next few weeks.

---
